app/components/card/card.twig

```diff
@@ -0,0 +1,18 @@
+{% macro default() %}
+{% import '../button/button.twig' as button %}
+{% import '../rating/rating.twig' as rating %}
+
+  <li class="card">
+    <img class="card__thumb" src="//lorempixel.com/240/120">
```

Comment:
Failed to load resource: net::ERR_CONNECTION_RESET: https://floidtv-pr-31.herokuapp.com/